packages/rnv/src/adapter/index.ts

```diff
@@ -17,7 +16,6 @@ export const withDefaultRNVBabel = (cnf: any) => ({
 
 export const withRNVBabel = (cnf: any) => (api: any) => {
     if (process.env.RNV_ENGINE_PATH && !fs.existsSync(process.env.RNV_ENGINE_PATH)) {
-        logInfo(`Path to engine cannot be resolved: ${process.env.RNV_ENGINE_PATH}. Will use default one`);
```

Comment:
@mihaiblaga89 can you elaborate on this change?
if issue is with actual "prevent withRnv functions to generate new context" this is not fixing the culprit so we need to dive deeper this can happen anywhere